scripts/openstack-crds.sh

```diff
@@ -8,6 +8,8 @@ function extract_crds {
     done
 }
 
+export REGISTRY_AUTH_FILE=$HOME/auth.json
```

Comment:
I think we should remove this from the PR. I'm not seeing this issue in my environment. Also, this seems like something that would be better set/managed in the users $HOME directory environment not hard coded like this.